### ISSUE TITLE
Changing the vis picker query should zoom to the top

### ISSUE THREAD

**Issue by ryantxu** [grafana/grafana#15956]
Using latest from master...

If you are scrolled down, and enter text in the query window, the results can be hidden:
![image](https://user-images.githubusercontent.com/705951/54241075-4655cc80-44dd-11e9-8458-ee6271326c3d.png)

**Comment by torkelo** [comment]
It is scrolled to the top for me when I open it. If you scroll while it's open I think you have to scroll back up again.

**Comment by ryantxu** [comment]
really?  I have found myself confused by this pretty often, and I should be used to it :)  Perhaps if the vis picker closed after selection.

To reproduce... use a small(ish) screen.  I am on a 15in mac.

1. Choose a visualization, I picked table:
![image](https://user-images.githubusercontent.com/705951/54291984-a6468480-456a-11e9-8d04-195c59d426db.png)

the vis changes, the icon changes color, but no options show up.

2. Scroll down to change the settings:
![image](https://user-images.githubusercontent.com/705951/54292124-dee65e00-456a-11e9-90b1-9902237ba9ff.png)


3. Now I want to change to another vis.  The query windows is still open, so I type: 'te' hoping to get the text2 panel.  but the view does not change until I scroll up.
